client/app/components/cards-list/CardsList.tsx

```diff
@@ -0,0 +1,73 @@
+import { includes, isEmpty } from "lodash";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import Input from "antd/lib/input";
+import Link from "@/components/Link";
+import EmptyState from "@/components/items-list/components/EmptyState";
+
+import "./CardsList.less";
+
+export interface CardsListItem {
+  title: string;
+  imgSrc: string;
+  onClick?: () => void;
+  href?: string;
+}
+
+export interface CardsListProps {
+  items?: CardsListItem[];
+  showSearch?: boolean;
+}
+
+function ListItem(item: CardsListItem, keySuffix: string) {
+  return (
+    <Link key={`card${keySuffix}`} className="visual-card" onClick={item.onClick} href={item.href}>
+      <img alt={item.title} src={item.imgSrc} />
+      <h3>{item.title}</h3>
+    </Link>
+  );
+}
```

Comment:
Overall code looks good, just a little last thing: let's convert this to a component, because it already looks like a component, just need to change arguments to props, and update usage

Reply:
Fixed in c39f254b2bef2f5962039ef8e1e60fa697d0a53e

Thanks for the review 😃